src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -194,18 +194,54 @@ export function start(manifest) {
 
 The following methods are provided:
 
-##### `app.render(request, { routeData, locals })`
+##### `app.render(request: Request, options?: RenderOptions)`
 
 This method calls the Astro page that matches the request, renders it, and returns a Promise to a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) object. This also works for API routes that do not render pages.
 
 ```js
 const response = await app.render(request);
 ```
 
-The method accepts a mandatory `request` argument, and an optional argument for [`routeData`](/en/reference/integrations-reference/#routedata-type-reference) and [`locals`](/en/reference/api-reference/#astrolocals).
+##### `RenderOptions`
+
+The `app.render()` method accepts a mandatory `request` argument, and an optional argument for [`clientAddress`](/en/reference/api-reference/#astroclientaddress),[`locals`](/en/reference/api-reference/#astrolocals), [`routeData`](/en/reference/integrations-reference/#routedata-type-reference).
+
+###### `addCookieHeader`
+
+Whether to automatically add all cookies written by `Astro.cookie.set()` to the response headers.
+When set to `true`, they will be added to the `Set-Cookie` header of the response as comma-separated key=value pairs. You can use the standard `response.headers.getSetCookie()` API to read them individually.
```

Comment:
```suggestion
When set to `true`, they will be added to the `Set-Cookie` header of the response as comma separated key-value pairs. You can use the standard `response.headers.getSetCookie()` API to read them individually.
```
I'm going to need a @Yan-Thomas check in aisle 5!

- comma separated: no hypen?
- key-value: hypen?
- "comma separated key-value pairs" no comma?

Reply:
> I'm going to need a @Yan-Thomas check in aisle 5!
> 
> * comma separated: no hypen?
> * key-value: hypen?
> * "comma separated key-value pairs" no comma?

If I may, I believe it should be: 
``they will be added to the `Set-Cookie` header of the response as comma-separated key-value pairs.``

Reply:
We could just be less specific. We are not manually performing this step, a standard API does it for us.